### DISCUSSION TITLE
Custom views [Apr 14]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1820] [Discord Threads]
> 🙋‍♂️ **acropolis11**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### Not yet, could you please let us know your usecase   
####    
#### is that like a page-designer ?  
#  
> 🙋‍♂️ **acropolis11**  
#### Yeah ofc .. looking to highlight and format different rows based on certain values  
#  
> 🙋‍♂️ **acropolis11**  
#### E.g highlight a column red if a value is below 1000  
#  
> 💁 **o1lab**  
#### ah got it  
#  
> 🙋‍♂️ **acropolis11**  
#### Is this a feature currently ?  
#  
> 💁 **o1lab**  
#### I dont think there has been a feature request like this before - you can pls raise it in our github  
#  
> 🙋‍♂️ **acropolis11**  
#### Ofc  
#  
> 🙋‍♂️ **acropolis11**  
#### Loving the project btw  
#  
> 🙋‍♂️ **acropolis11**  
#### Wishing the best 🫡  
#  
> 💁 **o1lab**  
#### thank you  
#  
> 💁 **o1lab**  
#### your encouragement helps  🙂  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/963809898716626964)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
